### PR TITLE
Removes Bulldogs from roundstart syndicate ship

### PR DESCRIPTION
**Pull request by LanCartwright**
Yeah you're gonna have to buy your guns now.
Don't worry if you need another weapon there's always the PDW.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
>making ops buy 5 guns+ammo without adjusting their TC at all

For what purpose. They just lost 10-20 roundstart TC from a bugfix as well

**Comment by LanCartwright** [comment]
As discussed you didn't really like the idea of the PDW.

I'm thinking of giving back a few (~5) uplinks with 10 TC but I don't think we understand the butterfly effect of juicing up ops with another ~50 tc. Not now atleast.

ideally the PDW should be added here because the PDW does not have 60 brute rounds and the versatility of the shotgun. 
In fact it actually deals more damage than the bulldog, but not to armoured targets.

**Comment by Bawhoppen** [comment]
wait for the other one to be merged first since you're so sure this is the right choice despite criticisms from many people

**Comment by KorPhaeron** [comment]
>80% winrate at lowpop
>30% winrate at highpop
>thinks removing bulldogs will somehow fix this

**Comment by Zilenan91** [comment]
Yeah no this is awful without increasing TCs

**Comment by LanCartwright** [comment]
Problem is @Zilenan91, in a 60 player round with 160 tc,you cannot inject another 50 tc like that with a snap of a finger.

**Comment by Zilenan91** [comment]
Yes you can, that's 60 freaking people. If people are in an even remotely validhuntey mood that day the Ops are gonna be swimming in bodies.

**Comment by KorPhaeron** [comment]
@LanCartwright

This is why I reccomend you slow down and wait till post freeze so you can make further adjustments/have a series of test merges etc. Nuke op scaling is gonna be tricky as hell and it's gonna suck if you only have 9 days to iron it out before its locked off for a month.

Voucher for a free bundle would probably be the most interesting/least disruptive though you'd run into people figuring out the best gun and ignoring the rest

**Comment by KorPhaeron** [comment]
PDW could still fit in with the voucher system by having it give you extra ammo vs the others or maybe some weight class shennanigans

**Comment by Bawhoppen** [comment]
Like I said before, we already have PDWs technically

**Comment by Jalleo** [comment]
I shall keep saying this we need scaled starting equipment for nuke ops.

**Comment by Lumipharon** [comment]
Voucher system yo.
Make is so that each op chooses a weapon in their uplink, and the weapon they choose determines the amount of tc's they have available for other shit.
This means that the ops can't just go nogunz and get 7 mechs or whatever because they have assloads of tc.

ie: Net effect is ops are better off in total amount of gear they get, but can't minmax and have loads of borgs and/or mechs.

**Comment by Steelpoint** [comment]
Just to say that the round start addition of Auto Rifles was during the time when Op win/lose rates were roughly even, or at least where not horribly one sided. I even took a screenshot on the 11th of October just as we removed the Rifles.

http://i.imgur.com/PFTqU2C.png (So not to clutter the convo with a massive image)

Moving back on topic, I agree overall with the removal of the round start Bulldogs, as their presence simply made getting any other weapon a bad idea when you already got a free powerful gun at round start.

About the issue of "muh TC's", just bloody give all Op uplinks a extra 10 TC's.

If Ops want to run a no-gun gimmick then more power to them, at best they may be able to afford maybe barely two mechs.

**Comment by Amemelius** [comment]
Give every operative 15 TC on their individual radios or this is just a huge nerf.

**Comment by tkdrg** [comment]
Yeah, I think it makes sense. Guns are cheaper now, so just give them maybe even 10 extra TC per ops (Which is basically what a voucher would do anyway), keep the stechkin, no need for PDWs IMO.

**Comment by Lumipharon** [comment]
yeah just give them more tc's and see what happens. even if it ends up unbalanced, it will be better then bullcuck every day.

**Comment by Steelpoint** [comment]
Ultimantly for better or worse this will serve to increase viarity in Op weapon choices, so instead of it being Bulldog galore there's at least the chance Ops may chose any weapon they like.

**Comment by LanCartwright** [comment]
15 TC more?

Do you not any idea of the imbalance this would cause by inflating the TC by juicing in 75 fucking telecrystals in?

Think chaos theory and butterfly effect. 
What would come of adding 80 players worth of telecrystals to roundstart?

75 TC+
100 from op radios.
20 from minimum nuke op.

that's 195 TC on a 20 crew round - I'd have to have the special uplinks banned in lowpop completely to prevent Mauler (140 tc) and 5 c20r (50 tc) combos.

75+100+60
235 TC on a 60 player round. I mean I don't see how well it can go from here.

**Comment by Steelpoint** [comment]
Adding 20 extra TC's in a 80 player round would actually be a good thing, considering how abysmally low Op win rates on very high pop are. 

At the end of the day we're just adding 15 to 20 extra TC's to the Ops, we can doomsay all we want but at the end of it its just that, speculation. 

I still say go for it, worst case scenario we can revert the 20 extra TC addition.

**Comment by tkdrg** [comment]
You're removing 40 TC's worth of free equipment from the ops with the bulldog removal. Under that perspective, giving each ops 5 extra TC isn't that big of a deal.

**Comment by Zilenan91** [comment]
Just give them about 75 more. That'll cover the costs of the bulldogs.

**Comment by LanCartwright** [comment]
I'll finish this tomorrow. If this isn't enough I'll throw in the towel and say the PDW is needed.

Which gives me food for thought; since Op challenge requires some strategic spending, I was considering giving them more weaponry (for free) so they're able to focus on challenge equipment.

**Comment by Bawhoppen** [comment]
thats why I like the voucher idea, so they have to get a gun and can't spend the extra 75 TC on borgs and other shit

**Comment by Lumipharon** [comment]
Don't add the pdw. They already have the perfectly functional stechkin. Just give them more tc's and get this shit merged. You can change it more later if needed.

**Comment by Steelpoint** [comment]
Merge when?

**Comment by Zilenan91** [comment]
They don't have more TC yet

**Comment by KorPhaeron** [comment]
>removing half the starting TC
>merge when

Never hopefully

**Comment by tkdrg** [comment]
Just give them more TC to compensate for the bulldogs, if they want to spend all TC on 5 c20rs then that's fine, they don't get any borgs or emags etc

**Comment by KorPhaeron** [comment]
>take away 5 guns+ammo
>still only enough tc added to buy 3 guns

File: code/game/objects/items/devices/uplinks.dm [hunk 1]
```diff
@@ -193,6 +193,7 @@ var/list/world_uplinks = list()
 /obj/item/device/radio/uplink/New()
 	hidden_uplink = new(src)
 	icon_state = "radio"
+	hidden_uplink.uses = 30
```
Comment (tkdrg) [inline code/game/objects/items/devices/uplinks.dm]: Don't do this here, I told you not to do it, and it won't even work. Do it in nuclear.dm post_equip(), chance tc to 30
https://github.com/tgstation/-tg-station/blob/master/code/game/gamemodes/nuclear/nuclear.dm#L313 this line

**Comment by Zilenan91** [comment]
Need to add 80 TC at the very least so they can buy the guns they want. Who cares if they buy mechs, mechs get shit on by the sheer numbers of the crew, the AI, and the Ion rifle.

**Comment by tkdrg** [comment]
Well, let's give it a try with 50 extra TC. If it's too little, we can easily raise it to 75 later.

**Pull request merged**